src/identities.js

```diff
@@ -3,9 +3,9 @@ const Identity = require('./identity')
 const IdentityProvider = require('./identity-provider-interface.js')
 const OrbitDBIdentityProvider = require('./orbit-db-identity-provider')
 const Keystore = require('orbit-db-keystore')
-
-const type = 'orbitdb'
-const identityKeysPath = './orbitdb/identity/identitykeys'
+const LRU = require('lru')
+const identityKeysPath = './orbitdb/identity/keys'
```

Comment:
Any issues with backward compatibility here? If people created identity keys with the old version and then upgraded, would this be able to find the old keys?

Reply:
They would need to pass in the old path in the options as the `identityKeysPath` but it would not be detected automatically